modules/Kconfig.mcuboot_bootutil

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+
+menuconfig MCUBOOT_BOOTUTIL_LIB
+	bool "mcuboot utility library"
+	help
+	  Enable mcuboot utility library which implements functions
+	  requret by the chainnloaded application and the MCUBoot.
+
+if !MCUBOOT && MCUBOOT_BOOTUTIL_LIB
+module = MCUBOOT_BOOTUTIL
```

Comment:
@tejlmand Is there possible to move this declaration to mcuboot repo. I have been trying to do so but couldn't menage to move it - even i was able to include a new Kconfig, it was not visible in kconfig processing results.

Reply:
I found another way, I made this able to be disabled by MCUBoot.

Reply:
I see that :+1: 
Sorry for the delay for getting to this.

I have placed a comment in the Kconfig for you to consider, though it's not blocking.